spec/models/profile_spec.rb

```diff
@@ -347,6 +347,50 @@
       expect(active_profile.verified_at).to be_present
     end
 
+    context 'when a user creates a biometric comparision profile' do
+      context 'when the user has an active profile' do
+        it 'creates a biometric upgrade record' do
+          profile.activate
+          biometric_profile = create(
+            :profile,
+            :biometric_proof,
+            user: user,
+          )
+
+          biometric_profile.activate
+
+          expect(SpUpgradedBiometricProfile.count).to eq(1)
```

Comment:
could make the expected side effect clearer with syntax like this?
```suggestion
          expect { biometric_profile.activate }.to(change { SpUpgradedBiometricProfile.count }).by(1))
```